eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -473,8 +470,6 @@ public function updateVersion($contentId, $versionNumber, Request $request)
 
         try {
             $this->repository->getContentService()->updateContent($versionInfo, $contentUpdateStruct);
-        } catch (ContentValidationException $e) {
-            throw new BadRequestException($e->getMessage());
         } catch (ContentFieldValidationException $e) {
             throw new BadRequestException($e->getMessage());
```

Comment:
don't we need to somehow pass on info from `getFieldErrors` to have parseable info for REST client, or is it supposed to parse the raw error message? /cc @dpobel